packages/venia-concept/src/actions/checkout/__tests__/asyncActions.spec.js

```diff
@@ -502,6 +510,6 @@ describe('formatAddress', () => {
         const values = { region_code: '|||' };
         const shouldThrow = () => formatAddress(values, countries);
 
-        expect(shouldThrow).toThrow('region');
```

Comment:
It seems like there should be two separate test cases here, one for `region` and one for `state`.

Reply:
https://github.com/magento-research/pwa-studio/pull/591#discussion_r243028782